lib/mapObjects/CGHeroInstance.cpp

```diff
@@ -150,6 +150,11 @@ bool CGHeroInstance::isCoastVisitable() const
 	return true;
 }
 
+bool CGHeroInstance::isBlockedVisitable() const
```

Comment:
I think this should not be needed if blockvisit is properly assigned in constructor...